src/commands/run-build.yml

```diff
@@ -10,6 +10,14 @@ parameters:
       in the plan returned by buildfile.m as well as all the tasks on which they depend.
     type: string
     default: ""
+  build-options:
+    description: >
+      MATLAB build options, specified as a list of options separated by spaces. The command
+      supports the same options that you can pass to the `buildtool` command when running a
+      MATLAB build. For more information, see
+      https://in.mathworks.com/help/matlab/ref/buildtool.html.
```

Comment:
Looking good, Kapil. Just noticed this: 
Why are we using in.mathworks.com instead of www.mathworks.com?

Reply:
Oops, it would have been a result of me copy pasting the visited link (from Bangalore, India). Good catch, fixing it!